### PR TITLE
feat: Add Metal L2 and Metal L2 testnet

### PR DESCRIPTION
**Pull request by squdgy**
**Description**

Adding Metal L2 and Metal L2 testnet chains.

**Tests**

N/A This change is config only

**Additional context**

N/A

**Metadata**

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ We currently accept tokens on the following chains:
 - `lisk`
 - `lisk-sepolia`
 - `redstone`
+- 'metall2'
```
Comment (pbatmetal) [inline README.md:122]: 'metall2' should be `metall2`

File: README.md [hunk 2]
```diff
@@ -119,6 +119,8 @@ We currently accept tokens on the following chains:
...
+- 'metall2-sepolia'
```
Comment (pbatmetal) [inline README.md:123]: 'metall2-sepolia' should be `metall2-sepolia`
replace ' with `

**Review by pbatmetal — COMMENTED** [review]
LGTM except docs update mentioned

**Review by pbatmetal — APPROVED** [review]

File: src/chains.ts [hunk 3]
```diff
@@ -155,10 +177,12 @@ export const L2_TO_L1_PAIR: Partial<Record<L2Chain, L1Chain>> = {
   mode: 'ethereum',
   lisk: 'ethereum',
   redstone: 'ethereum',
+  metall2: 'ethereum',
   'optimism-sepolia': 'sepolia',
   'base-sepolia': 'sepolia',
   'pgn-sepolia': 'sepolia',
   'lisk-sepolia': 'sepolia',
+  'metall2-sepolia': 'ethereum',
```
Comment (nitaliano) [inline src/chains.ts:185]: this should be `'metall2-sepolia': 'sepolia'`

File: src/chains.ts [hunk 4]
```diff
@@ -190,6 +214,10 @@ export const L1_STANDARD_BRIDGE_INFORMATION: Record<
       l2Chain: 'redstone',
       l1StandardBridgeAddress: '0xc473ca7E02af24c129c2eEf51F2aDf0411c1Df69',
     },
+    {
+      l2Chain: 'metall2',
+      l1StandardBridgeAddress: '0xAEb9E3dD0074A8dd19065f55235591c3Ac0c413E',
```
Comment (nitaliano) [inline src/chains.ts:219]: this doesn't match whats in the superchain-registry https://github.com/ethereum-optimism/superchain-registry/blob/10ff8b6e7b57cbd894d333d13c59c08bb3970cac/superchain/configs/mainnet/metal.toml#L49

File: src/chains.ts [hunk 5]
```diff
@@ -208,5 +236,9 @@ export const L1_STANDARD_BRIDGE_INFORMATION: Record<
       l2Chain: 'lisk-sepolia',
       l1StandardBridgeAddress: '0x1Fb30e446eA791cd1f011675E5F3f5311b70faF5',
     },
+    {
+      l2Chain: 'metall2-sepolia',
+      l1StandardBridgeAddress: '0xFA0EBa40F30338B50A08613Af2216f914Ae8a7B4',
```
Comment (nitaliano) [inline src/chains.ts:241]: same here https://github.com/ethereum-optimism/superchain-registry/blob/10ff8b6e7b57cbd894d333d13c59c08bb3970cac/superchain/configs/sepolia/metal.toml#L45

Comment (squdgy) [reply src/chains.ts]: @nitaliano I've fixed these issues. Thank you.

**Review by nitaliano — APPROVED** [review]
LGTM thanks getting metal setup!

**Pull request merged**